bin/yang_get_stats.py

```diff
@@ -441,43 +72,383 @@ def write_dictionary_file_in_json(in_dict, path, file_name):
         ('Packet Design', 'packetdesign.com'),
         ('Qosmos', 'qosmos.com')
     )
-
-    def print_attribution(name, domain):
-        if not name and not domain:
-            print()
+    YANG_PAGE_MAIN_PREFIX = 'YANGPageMain_'
+    IETF_YANG_PAGE_MAIN_PREFIX = 'IETFYANGPageMain_'
+    IETF_YANG_OUT_OF_RFC_PREFIX = 'IETFYANGOutOfRFC_'
+
+    def __init__(self, args: argparse.Namespace, config: ConfigParser = create_config()):
+        self.args = args
+        self.debug_level = args.debug
+        self.web_private = config.get('Web-Section', 'private-directory')
+        self.backup_directory = config.get('Directory-Section', 'backup') + '/'
```

Comment:
I would prefer to avoid appending `'/'` at the end of the directory path.
Instead of this check all the places where `self.backup_directory` is used and try to use something like `os.path.join()` method.